test/integration/timestamp_generation_test.exs

```diff
@@ -28,7 +28,21 @@ defmodule Membrane.H264.TimestampGenerationTest do
   end
 
   @h264_input_file "test/fixtures/input-10-720p.h264"
+  @h264_input_timestamps [
```

Comment:
I believe that the profile there is `main`, so for the sake of consistency with the other attributes names we can call these attributes `@h264_input_file_main` and `@h264_input_timestamps_main`